claims_test.go

```diff
@@ -376,3 +377,56 @@ func TestValidateBoundClaims(t *testing.T) {
 		}
 	}
 }
+
+func Test_normalizeList(t *testing.T) {
+	tests := []struct {
+		raw        interface{}
+		normalized []interface{}
+		ok         bool
+	}{
+		{
+			raw:        []interface{}{"green", 42},
+			normalized: []interface{}{"green", 42},
+			ok:         true,
+		},
+		{
+			raw:        []interface{}{"green"},
+			normalized: []interface{}{"green"},
+			ok:         true,
+		},
+		{
+			raw:        []interface{}{},
+			normalized: []interface{}{},
+			ok:         true,
+		},
+		{
+			raw:        "green",
+			normalized: []interface{}{"green"},
+			ok:         true,
+		},
+		{
+			raw:        "",
+			normalized: []interface{}{""},
+			ok:         true,
+		},
+		{
+			raw:        42,
+			normalized: nil,
+			ok:         false,
+		},
+		{
+			raw:        nil,
+			normalized: nil,
+			ok:         false,
+		},
+	}
+	for _, tt := range tests {
+		normalized, ok := normalizeList(tt.raw)
+		if !reflect.DeepEqual(normalized, tt.normalized) {
+			t.Errorf("normalizeList() got = %v, want %v", normalized, tt.normalized)
+		}
+		if ok != tt.ok {
+			t.Errorf("normalizeList() got1 = %v, want %v", ok, tt.ok)
```

Comment:
Is this `got1` a typo or are you trying to have a different error message for this case?

Reply:
oops. I used the GoLand test case generator which is convenient but has some stuff that I usually change, or in this case forget to change.